### ISSUE TITLE
Export FunctionEvent type

### ISSUE THREAD

**Issue by Grmiade** [graphcool/graphcool-lib#11]
It would be great to export the FunctionEvent type in the definition file.
To use it to type `event` parameter.

```typescript
import { FunctionEvent } from 'graphcool-lib'

export default async (event: FunctionEvent) => {
  await new Promise(r => setTimeout(r, 50))
  return {
    data: {
      message: `Hello ${event.data.name || 'World'}`
    }
  }
}
```

**Comment by kbrandwijk** [comment]
Part of a WIP Pull request: https://github.com/graphcool/graphcool-lib/pull/9/commits/89322a6629fd8304552700bba881657d17451f0f

**Comment by marktani** [comment]
It has been released in `0.1.0`: https://github.com/graphcool/graphcool-lib/releases/tag/v0.1.0 🙂
